### PR TITLE
add bedmaster indexer documentation

### LINKED ISSUE #129: Document Bedmaster file indexer
PR https://github.com/aguirre-lab/ml4c3/pull/1278 introduces an awesome new functionality.

We should document how to connect to it and some simple examples of how to use it.

### PR DESCRIPTION
**Pull request by StevenSong**
closes #129 
documentation for https://github.com/aguirre-lab/ml4c3/pull/1278

### CODE REVIEW

**Review by raimonpv — APPROVED** [review]

**Review by erikr — APPROVED** [review]
Excellent documentation, thank you! Will be helpful for the future.

Re:

> The indexer is setup to run every night at midnight on `everest`

Can you please add a description of how it is set up to run regularly, and how one might adjust that? Is it a `cron` job?

File: Bedmaster-indexer.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+--username $USERNAME \
+--password $PASSWORD \
+--batch_size 100 \
+--num_workers 4 \
+--index_file /media/ml4c3/bedmaster-index.csv
+```
+
+### Database details
+
+The database is running on `mithril` at port `5432`.
```
Comment (erikr) [inline Bedmaster-indexer.md:49]: Can you add simple instructions (or link to relevant documentation) to help users access the db? e.g. intro to postgres.

Comment (StevenSong) [reply Bedmaster-indexer.md]: done

**Comment by StevenSong** [comment]
> Can you please add a description of how it is set up to run regularly, and how one might adjust that? Is it a `cron` job?

it is a cron job, added to docs

**Pull request merged**